client/app/src/App.js

```diff
@@ -1,14 +1,75 @@
-import Amplify from 'aws-amplify';
 import React from 'react';
 import { Platform, StatusBar, StyleSheet, View } from 'react-native';
+import { Provider } from 'react-redux';
+import { PersistGate } from 'redux-persist/integration/react';
+import Amplify from 'aws-amplify';
+import { withAuthenticator } from 'aws-amplify-react-native';
 import EStyleSheet from 'react-native-extended-stylesheet';
 import { DefaultTheme, Provider as PaperProvider } from 'react-native-paper';
-import { withAuthenticator } from 'aws-amplify-react-native';
-
+import { persistor, store, actions } from './state';
+import concertApi from './api/concertApi';
+import NavigationService from './navigation/NavigationService';
 import AppNavigator from './navigation/AppNavigator';
 import layout from './constants/Layout';
 import config from './config';
 
+// ** Event listeners ** //
+const handleStageNavigation = stageId => {
+  console.log('in handleStageNavigation');
+  const currentState = store.getState();
```

Comment:
`currentState` isn't being used.
```suggestion
  const { choiceType } = store.getState();
```